go/tendermint/abci/mux.go

```diff
@@ -594,7 +594,7 @@ func (mux *abciMux) CheckTx(req types.RequestCheckTx) types.ResponseCheckTx {
 	if err := mux.executeTx(ctx, req.Tx); err != nil {
 		return types.ResponseCheckTx{
 			Code: api.CodeTransactionFailed.ToInt(),
-			Info: err.Error(),
+			Log:  err.Error(),
```

Comment:
Since you changed this, don't forget to also change `broadcastTx` (around line 465) which still uses `GetInfo()`.